src/test/java/teammates/test/cases/action/InstructorCourseRemindActionTest.java

```diff
@@ -45,6 +45,7 @@ public void testExecuteAndPostProcess() throws Exception {
         };
 
         InstructorCourseRemindAction remindAction = getAction(submissionParams);
+        remindAction.setCurrentUrl("/page/instructorCourseEditPage?");
```

Comment:
This isn't necessary. This is why:
https://github.com/TEAMMATES/teammates/blob/847de291e580ffbe3f9c50b329bb29acce5818a7/src/main/java/teammates/ui/controller/InstructorCourseRemindAction.java#L70-L85

you can verify this by deleting this added line, the test would still pass

Reply:
That's true. I have removed this line now. Thank you!